### PR TITLE
Fix formatting issue

### PR DESCRIPTION
**Pull request by nsharma-git**
# Description

This PR fixes a few formatting issue for rbac_replicator task

Fixes # (ANSIENG-998)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

```
TASK [confluent.kafka_connect_replicator : Configure RBAC on Replicator configuration] ***
included: /Work/repos/ansible_collections/confluent/platform/roles/confluent.kafka_connect_replicator/tasks/rbac_replicator.yml for kafka-connect-replicator1

TASK [confluent.kafka_connect_replicator : Create SSL Certificate Directory] ***
--- before
+++ after
@@ -1,4 +1,4 @@
 {
     "path": "/var/ssl/private/kafka_connect_replicator",
-    "state": "absent"
+    "state": "directory"
 }

changed: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Copy in MDS Public Pem File] ********
changed: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User SystemAdmin on Kafka cluster] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User SystemAdmin on Replicator Cluster] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User ResourceOwner on Connect Topics] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Read on Command Topic] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Write on Command Topic] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Manage on Command Topic] ***
skipping: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User SystemAdmin on Kafka cluster] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User SystemAdmin on Replicator Cluster] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User ResourceOwner on Connect Topics] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Read on Command Topic] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Write on Command Topic] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Developer Manage on Command Topic] ***
ok: [kafka-connect-replicator1]

TASK [confluent.kafka_connect_replicator : Grant Confluent Replicator User Security Admin on Destination cluster] ***
ok: [kafka-connect-replicator1]

```

**Test Configuration**:
` molecule --debug converge -s rbac-kafka-connect-replicator-kerberos-mtls-custom-rhel`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Review by utkarsh5474 — COMMENTED** [review]
Hey @nsharma-git 
There a couple of more files where there are formatting issues of extra `}}` like this one `"{{kafka_connect_replicator_kafka_cluster_id}}"}}` 
The files I could find - 
```
roles/confluent.kafka_connect/tasks/rbac.yml
roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_consumer.yml
roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_producer.yml
```
Should we add this in the current PR

**Comment by nsharma-git** [comment]
> Hey @nsharma-git There a couple of more files where there are formatting issues of extra `}}` like this one `"{{kafka_connect_replicator_kafka_cluster_id}}"}}` The files I could find -
> 
> ```
> roles/confluent.kafka_connect/tasks/rbac.yml
> roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_consumer.yml
> roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_producer.yml
> ```
> 
> Should we change these in the current PR?

Thanks @utkarsh5474.
 I think it make sense to have these changes in this PR itself. I'll update this PR shortly.

**Review by utkarsh5474 — CHANGES_REQUESTED** [review]
I believe, most of the changes around double parenthesis will need to be changed as per this - https://docs.confluent.io/platform/current/security/rbac/mds-api.html#post--security-1.0-principals-principal-roles-roleName-bindings

**Comment by nsharma-git** [comment]
> I believe, most of the changes around double parenthesis will need to be changed as per this - https://docs.confluent.io/platform/current/security/rbac/mds-api.html#post--security-1.0-principals-principal-roles-roleName-bindings

Sorry, didn't get it. What changes you are proposing?

**Comment by nsharma-git** [comment]
> I believe, most of the changes around double parenthesis will need to be changed as per this - https://docs.confluent.io/platform/current/security/rbac/mds-api.html#post--security-1.0-principals-principal-roles-roleName-bindings

Good catch @utkarsh5474 
Can you please have a look on the updated PR

File: roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_producer.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@
       {
         "scope": {
           "clusters": {
-            "kafka-cluster":"{{kafka_connect_replicator_producer_kafka_cluster_id}}"}},
+            "kafka-cluster":"{{kafka_connect_replicator_producer_kafka_cluster_id}}",
```
[the remark this answers is not in the document]
Comment (nsharma-git) [reply roles/confluent.kafka_connect_replicator/tasks/rbac_replicator_producer.yml]: Done!

**Review by utkarsh5474 — APPROVED** [review]
Thanks!

**Pull request merged**